2019-10-12 16:17:57,276 ERROR: Exception on /profile [GET] [in /Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/views.py", line 163, in dispatch_request
    return meth(*args, **kwargs)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/app/main/routes.py", line 43, in get
    return render_template('main/profile.html', form=form, links=links)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/app/templates/main/profile.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/app/templates/main/base.html", line 42, in top-level template code
    {% block content %}
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/app/templates/main/profile.html", line 13, in block "content"
    {{ wtf.form_field(form.messenger_type) }}
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/jinja2/runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/wtf.html", line 123, in template
    {{field(class="form-control", **kwargs)|safe}}
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/wtforms/fields/core.py", line 155, in __call__
    return self.meta.render_field(self, kwargs)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/wtforms/meta.py", line 56, in render_field
    return field.widget(field, **render_kw)
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/wtforms/widgets/core.py", line 323, in __call__
    for val, label, selected in field.iter_choices():
  File "/Users/zverev/PycharmProjects/learn_python/affiliate/venv/lib/python3.7/site-packages/wtforms/fields/core.py", line 454, in iter_choices
    for value, label in self.choices:
TypeError: 'NoneType' object is not iterable
